relay/sources/relayd/src/configuration.rs

```diff
@@ -425,6 +427,8 @@ mod tests {
                 nodes_certs_file: PathBuf::from("tests/keys/nodescerts.pem"),
                 node_id: "root".to_string(),
                 listen: "127.0.0.1:3030".parse().unwrap(),
+                core_threads: None,
+                blocking_threads: 100,
```

Comment:
how was 100 chosen ? isn't it a lot ?

Reply:
It's the default, and this pool is only used for io blocking operations. We can move to something like 50, it would probably be enough for most usecases.